htdocs/core/triggers/interface_50_modTicket_TicketEmail.class.php

```diff
@@ -279,7 +279,10 @@ public function runTrigger($action, $object, User $user, Translate $langs, Conf
 						if ($mailfile->error) {
 							dol_syslog($mailfile->error, LOG_DEBUG);
 						} else {
-								  $result = $mailfile->sendfile();
+							$result = $mailfile->sendfile();
+							// update last_msg_sent date
```

Comment:
May be you should add a test on result on ->sendfile to update only if email sending is ok ?

Reply:
Thanks, I also added a test on ticket->sendTicketMessageByEmail() .